gridt/models/movement_user_relation.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime
+from sqlalchemy import Column, Integer, DateTime, ForeignKey, String
+from sqlalchemy.orm import relationship
+
+from gridt.db import Base
+
+class MovementUserRelation(Base):
```

Comment:
The difference between Movement user relation and movement user association isn't very clear

Reply:
I agree! I discussed with Yori about rename movement user association to UserUserMovementLink (because it is an actual link in the network). I thought about it more today maybe another name could be LeaderFollowerLink

Reply:
But I think this is better done in a new pull request